tests/wpt/mozilla/tests/mozilla/custom_auto_rooter.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<title>Types using Custom Auto Rooter work correctly</title>
+<head>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<script>
+test(function() {
+  function assert_array_equals(lhs, rhs) {
```

Comment:
This should already exist: http://web-platform-tests.org/writing-tests/testharness-api.html

Reply:
Fixed, thanks!